tests/migration_test.go

```diff
@@ -1231,11 +1232,8 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				By("Waiting for VMI to disappear")
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 			},
-				table.Entry("[test_id:2653] with default migration configuration", nil, v1.MigrationPreCopy),
-				table.Entry("[QUARANTINE][test_id:5004] with postcopy", &v1.MigrationConfiguration{
-					AllowPostCopy:           pointer.BoolPtr(true),
-					CompletionTimeoutPerGiB: pointer.Int64Ptr(1),
-				}, v1.MigrationPostCopy),
+				table.Entry("[test_id:2653] with default migration configuration", v1.MigrationPreCopy),
+				table.Entry("[QUARANTINE][test_id:5004] with postcopy", v1.MigrationPostCopy),
```

Comment:
Let's please separate this test into its own Context, outside of this NFS tests.
The will be some duplication but at least it will have its own BeforeEach and its own settings.

Reply:
Done. Thanks!

Reply:
Also, seems like we have a "migration postcopy" context.. What's more fitting than that? :)